src/role_scout.js

```diff
@@ -13,153 +13,226 @@ roles.scout.settings = {
   maxLayoutAmount: 1
 };
 
-function onBorder(creep) {
+roles.scout.execute = function(creep) {
+  if (creep.memory.skip === undefined) {
+    creep.memory.skip = [];
+  }
+
+  creep.notifyWhenAttacked(false);
```

Comment:
This always costs at least 0.2 cpu. Maybe we can invoke it only once per creep?

Reply:
was previously here but can add config check with a flag in memory for do it only once.
maybe better in another PR